pipeline/src/resources/ColorAnnotator.py

```diff
@@ -0,0 +1,59 @@
+
+import tornado.web
+import socket
+import tornado.ioloop
+import json
+
+
+class Annotator(tornado.web.RequestHandler):
```

Comment:
We might want to put this in its own file, since this class will hopefully be used by many other annotators.

Reply:
I agree here as well.